src/Wallabag/ApiBundle/Controller/EntryRestController.php

```diff
@@ -286,6 +286,51 @@ public function patchEntriesAction(Entry $entry, Request $request)
     }
 
     /**
+     * Reload an entry.
+     * A response with HTTP Status 400 will be return if we weren't able to update the content (because it hasn't changed or we got an error).
```

Comment:
> 400 Bad Request
> The server cannot or will not process the request due to an apparent client error (e.g., malformed request syntax, too large size, invalid request message framing, or deceptive request routing).

> because it hasn't changed

It's not an error, so I don't know if status 400 is the good one.